frontend/simple/views/utils/validators.js

```diff
@@ -1,5 +1,13 @@
+import { HashableGroupProposal } from '../../../../shared/events.js'
+
 export const nonWhitespace = (value: string) => /^\S+$/.test(value)
 
 export const decimals = (digits: number) => (value: number) => Number.isInteger(value * Math.pow(10, digits))
 
-export const votingType = (type: String) => ['mincome', 'rule', 'member'].includes(type)
+export const votingType = (type: String) => [HashableGroupProposal.TypeInvitation, HashableGroupProposal.TypeRemoval, HashableGroupProposal.TypeIncome, HashableGroupProposal.TypeChangeThreshold, HashableGroupProposal.TypeApprovalThreshold, HashableGroupProposal.TypeRemovalThreshold].includes(type)
```

Comment:
I'm wondering... These 2 validators are really specific for Voting props component only. Is this the right place for it? Maybe they should be next to the component itself `Voting/validators.js`, otherwise soon this will get huge.

Reply:
yes, it might be a better place there 🤔